pkg/config/application.go

```diff
@@ -66,6 +67,55 @@ func NewApplicationGroup(application *Application, mspConfig *mb.MSPConfig) (*cb
 	return applicationGroup, nil
 }
 
+// AddAnchorPeer adds an anchor peer to an existing channel config transaction.
+// It must add the anchor peer to an existing org and the anchor peer must not already
+// exist in the org.
+func AddAnchorPeer(config *cb.Config, orgName string, newAnchorPeer *AnchorPeer) error {
+	if _, orgExists := config.ChannelGroup.Groups["Application"].Groups[orgName]; !orgExists {
```

Comment:
1. Here we use `"Application"` and below we use `ApplicationGroupKey`. It's inconsistent.
2. Why aren't we holding the result instead of just getting it again on line 78?